bin/regressive_test.sh

```diff
@@ -14,6 +14,17 @@ fi
 
 ./recompile.sh
 . ./install_dbtoaster.sh
+# check if scala installed
+if hash scala 2>/dev/null; then
+    version=$(scala -version 2>&1 | awk -F " " '{print $5}') 
+    if [[ $version != "2.10"* ]]; then
```

Comment:
@akathorn the code is compiled with 2.10. I have not tested it with 2.11. I follow the requirement of DBToaster installation guide: http://www.dbtoaster.org/index.php?page=docs . It is good to know that 2.11 works. I will remove this check for now. Thanks.